README.md

```diff
@@ -291,6 +292,62 @@ sdk.isAuthenticated({ loadFileSystem: false })
 const fs = await sdk.loadFileSystem()
 ```
 
+# Apps API
+The sdk also exposes methods to interact with the apps associated with the user. This API must be prefixed with `apps`
+- `apps.index`: A list of all of your apps and their associated domain names
+- `apps.create`: Creates a new app, assigns an initial subdomain, and sets an asset placeholder
+- `apps.deleteByURL`: Destroy app by any associated URL
```

Comment:
Good stuff 💪 

Can you move this entire section (Apps API) below "Versions" and "Customisation"?
"Versions" is part of the file system section above this, and "Customisation" is a bit more important.

Reply:
Thanks Nathan! Looks like you still need to do this change? Let me know if I didn't explain this properly 😳